broker-node/app/Clients/BrokerNode.php

```diff
@@ -291,18 +290,16 @@ public static function processChunks(&$chunks)
         }
     }
 
-    public static function filterUnattachedChunks($addresses, $chunks)
+    private static function filterUnattachedChunks($addresses, $chunks)
```

Comment:
Thanks for changing all the other methods to private.  This one we might need to keep public though?  I think CheckChunkStatus will need to use either this, or the verifyChunkMatchesRecord.  Speaking of which, I need to change that method to accept an array as well.   

We can leave it like this for now though, there's no cron task currently afaik.

Reply:
Reverted.